test/readme.md

```diff
@@ -263,7 +263,29 @@ The following steps will walk you through how to create and test a MongoDB Serve
 
 The following steps will walk you through how to start and test a load balancer.
```

Comment:
Can we add a note about using a server 5.2.0 or above?

Reply:
Done.

Reply:
Did you add this note anywhere?  @durran

Reply:
Yeah - forgot to push. It's there now.